commands/actions/publish.go

```diff
@@ -95,8 +95,20 @@ func buildFunc(cmd *cobra.Command, args []string) {
 	}
 	mustParseAndValidateTriggers(actions)
 
+	existsTsFiles, err := util.TsFilesExists(actions.Sources)
+	if err != nil {
+		logrus.Error(commands.Colorizer.Sprintf("Found error when finding *.ts files, Error: %s", commands.Colorizer.Bold(commands.Colorizer.Red(err))))
+		os.Exit(1)
```

Comment:
This is an optional, best effort check, I'd consider ignoring error here, just logging skip the check